### PR TITLE
Add a check for having a single argument

### PR DESCRIPTION
**Pull request by farcaller**
Signed-off-by: Vladimir Pouzanov <farcaller@gmail.com>

Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [x] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [x] Provide a title or release-note blurb suitable for the release notes.
- [x] Thanks for contributing!

This fixes a crash in `cilium bpf endpoint delete` when ran without arguments.

```release-note
Fix a crash in `cilium bpf endpoint delete` when ran without arguments.
```

### CODE REVIEW

**Review by ldelossa — APPROVED** [review]

**Comment by farcaller** [comment]
I'm slightly confused by the `dont-merge/needs-release-note-label` because there is one in the PR description. Any idea?

**Review by joamaki — APPROVED** [review]

**Comment by farcaller** [comment]
Ran gofmt and made the error message actually useful 😄

**Review by dylandreimerink — APPROVED** [review]

File: cilium/cmd/bpf_endpoint_delete.go [hunk 1]
```diff
@@ -18,6 +18,10 @@ var bpfEndpointDeleteCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
```
Comment (sayboras) [inline cilium/cmd/bpf_endpoint_delete.go:19]: just curious if you have tried the below :thinking: 

```
	Args:  cobra.ExactArgs(1),
	Run: func(cmd *cobra.Command, args []string) {
```

Comment (farcaller) [reply cilium/cmd/bpf_endpoint_delete.go:19]: I didn't! I should have remembered cobra had it.

**Review by sayboras — APPROVED** [review]
LGTM :100:

**Comment by gandro** [comment]
I don't think this needs a full CI run, since `cilium bpf endpoint delete` is never executed in in CI. Marking ready to merge.

**Pull request merged**